server/pulp/server/webservices/urls.py

```diff
@@ -15,6 +22,20 @@
 
 urlpatterns = patterns('',
     url(r'^v2/actions/login/$', LoginView.as_view(), name='login'),
+    url(r'^v2/consumer_groups/$', ConsumerGroupView.as_view(), name='consumer_group'),
+    url(r'^v2/consumer_groups/(?P<consumer_group_id>[^/]+)/$',
+        ConsumerGroupResourceView.as_view(), name='consumer_group_resource'),
+    url(r'^v2/consumer_groups/(?P<consumer_group_id>[^/]+)/actions/associate/$',
+        ConsumerGroupAssociateActionView.as_view(), name='consumer_group_associate'),
+    url(r'^v2/consumer_groups/(?P<consumer_group_id>[^/]+)/actions/unassociate/$',
+        ConsumerGroupUnassociateActionView.as_view(), name='consumer_group_unassociate'),
+    url(r'^v2/consumer_groups/(?P<consumer_group_id>[^/]+)/actions/content/(?P<action>[^/]+)/$',
+        ConsumerGroupContentActionView.as_view(), name='consumer_group_content'),
+    url(r'^v2/consumer_groups/(?P<consumer_group_id>[^/]+)/bindings/$',
+        ConsumerGroupBindingsView.as_view(), name='consumer_group_bind'),
+    url(r'^v2/consumer_groups/(?P<consumer_group_id>[^/]+)' + \
```

Comment:
flake8 says: "E502 the backslash is redundant between brackets" Since its within parenthes it should not require the backslash.